plugins/source/typeform/plugin/tables/form_responses.py

```diff
@@ -29,11 +30,13 @@ def __init__(self) -> None:
                 Column("variables", JSONType()),
                 Column("tags", JSONType()),
             ],
+            is_incremental=True,
         )
+        self._resolver = FormResponsesResolver(table=self)
 
     @property
     def resolver(self):
-        return FormResponsesResolver(table=self)
+        return self._resolver
```

Comment:
This is a dangerous anti-pattern we should be aware of. It was creating a brand new resolver every time the property was being referenced (note that by adding `@property`, you don't call `resolver()`, but just `resolver`).

Reply:
@marianogappa What's the risk of this usage, out of curiosity? Is it performance only?

Reply:
@dkeyer-twilio oh sorry, I should have been more clear. There's no performance issue.

- `resolver` here is acting as a `getter`, and one doesn't expect a getter to create a brand new object.
- Objects are stateful. One wouldn't normally expect to set some state on an object and then not have it later.
- In this case, the `.sync()` method calls `resolver.set_state_client(...)`, and this method would erase that state.

I think this pattern causes very unintuitive behaviour. I certainly lost an hour on it.